src/renderer/services/file-storage-service/test/file-storage-service.test.ts

```diff
@@ -60,6 +61,49 @@ describe("FileStorageService", () => {
   });
 
   describe("sendUploadChunk", () => {
+    class AxiosError extends Error {
+      public response: any;
+
+      constructor(response: any) {
+        super("test error");
+        this.response = response;
+      }
+    }
+
+    it("retries chunk when error occurs and server status indicates RETRY", async () => {
```

Comment:
No test of the error when the number of retries is exceeded.  Is that intentional?

Reply:
I didn't have a good way to test this that didn't slow the test suite down significantly